src/test/test_hs_service.c

```diff
@@ -289,6 +289,20 @@ helper_create_service(void)
   return service;
 }
 
+/* Helper: Deallocate a given service object, it's child objects and
```

Comment:
s/it's/its/

Reply:
Fixed - 151ce23dc22383ebc754931e288bc2ad604cece9.